diagnostic.md

```diff
@@ -5,74 +5,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+For a website to connect to the web which allows users to interact throught a site.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+I am not sure what is meant by 'layer'. the controller sends the information it got from the server to the model so that information from the database is sent back.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+I am not sure what is meant by 'layer'. The model gets a request from the controller to fetch data from the database by converting the ruby into sql and sends the data back to the controller.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Not sure, I was asking myself that the other day. We have just been using the controller to send json to the user instead of making it available to the views. I thought the views is what the user saw, but they can also see it with json?
 ```
```

Comment:
Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.